### DISCUSSION TITLE
Competition results!

### DISCUSSION THREAD

**Discussion by maxbannach** [esa/SpOC3#15] [Announcements]
Here is the global leaderboard of SpOC3. Congratulations to the winners from **Team HRI** 🥳.

| Rank | Team           | Global Score |
|:-----|:---------------|:-------------|
| 🥇   | Team HRI       | 78.444       |
| 🥈   | fcmaes         | 68.667       |
| 🥉    | Spacekangaroos | 66.555       |
| 4    | theEs          | 63.445       |
| 5    | ArgotecR&D     | 38.778       |
| 6    | jakeforsey     | 33.444       |
| 7    | Rehman K.      | 20.222       |
| 8    | MasterRoom     | 17.778       |
| 9    | tristan        | 16.667       |
| 10   | patrulek       | 7.222        |
| 11   | POPOGO         | 7            |
| 12   | iohprofiler    | 5.333        |

Stay tuned for new challenges on Optimize and our other platforms! We hope to see you again very soon for the next edition of SpOC 🖖

**Comment by CoolRunning** [comment]
Congratulations to Team HRI for a well deserved victory! Big thanks to all organizers for putting up such interesting challenges and all other competitors for giving their best.

**Reply by neinecke** [reply]
Hey Marcus,
Thanks for the congrats. Which Team do you belong to?

**Reply by CoolRunning** [reply]
Me and some friends are team Spacekangaroos. :)

**Reply by neinecke** [reply]
Hello Spacekangarooeans,
Nice to meet you and also congrats for the very good results on the torso problem. I'm looking forward to meet you in person at the ESA workshop and learn about your approach.

**Reply by dietmarwo** [reply]
Hello Spacekangaroos, 
this is team fcmaes (which is only one person, me). Congrats to your excellent performance. Specially your late engagement in interferometric and what you achieved with the large graph decomposition task is impressive. Hope to see your team in Leiden. One (fun-) fact about my own approach: I used the same optimization algorithm adapted for each task - even the cubes. Guess from which optimization library it is ;-) .
